oe_corporate_blocks.install

```diff
@@ -6,3 +6,47 @@
  */
 
 declare(strict_types = 1);
+
+use Drupal\Core\Config\Entity\ConfigEntityType;
+use Drupal\Core\StringTranslation\TranslatableMarkup;
+
+/**
+ * Install new 'Footer Link Section' config entity type.
+ */
+function oe_corporate_blocks_update_8301(&$sandbox): void {
+  \Drupal::entityDefinitionUpdateManager()->installEntityType(new ConfigEntityType([
```

Comment:
not sure why we need to redefine it here, we just need to install it no ?
can't we just do:
`$type_manager = \Drupal::entityTypeManager();`
`$type_manager->clearCachedDefinitions();`
`$entity_type = $type_manager->getDefinition('footer_link_section');`
` \Drupal::entityDefinitionUpdateManager()->installEntityType($entity_type);`
?

Reply:
https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EWPP-485?focusedCommentId=4428776&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-4428776